prv_accountant/privacy_random_variables/abstract_privacy_random_variable.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import numpy as np
+
+from abc import ABC, abstractmethod
+from scipy import integrate
+
+
+class NoRDPImplementation(NotImplementedError):
+    pass
```

Comment:
Why do we use this particular subclass for a `NotImplementedError` but not for `pdf` of `cdf`, for example?

Reply:
You're right - it's not really needed. I've removed it.